libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -394,13 +394,8 @@ protected void onAuthFlowComplete(TokenEndpointResponse tr) {
     private class SwapJWTForAccessTokenTask extends BaseFinishAuthFlowTask<LoginOptions> {
 
         @Override
-        protected TokenEndpointResponse performRequest(LoginOptions options) {
-            try {
```

Comment:
The `OAuth2.swapJWTForTokens` call throws a bunch of exceptions though. We need to catch them here, but instead of just logging them, we should call `onAuthFlowError` here.